keras_cv/models/mobilenet_v3.py

```diff
@@ -112,48 +112,44 @@ def depth(x, divisor=8, min_value=None):
     return new_x
 
 
-def HardSigmoid(name=None):
+def apply_hard_sigmoid(x, name=None):
     """The Hard Sigmoid function.
 
     Args:
+        x: input tensor
         name: string, layer label.
 
     Returns:
-        a function that takes an input Tensor representing a HardSigmoid layer.
+        the updated input tensor.
```

Comment:
(throughout) - it would be more accurate to say that this returns "the result of applying hard sigmoid to the input tensor"

Reply:
Actually, this looks consistent with the docs in other models, so feel free to ignore this for now and we can address it in all the models at once in #1500

Reply:
I'll take this up directly while solving #1500 then.